airbyte-db/db-lib/src/main/java/io/airbyte/db/jdbc/streaming/FetchSizeConstants.java

```diff
@@ -10,7 +10,7 @@ public final class FetchSizeConstants {
   // This size is not enforced. It is only used to calculate a proper
   // fetch size. The max row size the connector can handle is actually
   // limited by the heap size.
-  public static final double TARGET_BUFFER_SIZE_RATIO = 0.5;
+  public static final double TARGET_BUFFER_SIZE_RATIO = 0.6;
```

Comment:
curious, why not go higher?

if primary memory usage is due to this buffer, I think we can go as high as 80%. Usual java overhead is ~~10% so a 20% buffer seems good enough.

Reply:
I think we can keep ramping this up... but in an environment where I can't robustly test this, I want to go step-by-step